### ISSUE TITLE
Error in README.md with install instructions 

### ISSUE THREAD

**Issue by bluepuma77** [capacitor-community/sqlite#520]
**Describe the bug**
`README.md` states:
```
then add plugin to main `capacitor.config.json` file:
```
with code example:
```
import { CapacitorConfig } from '@capacitor/cli';
...
```
which is clearly **not** JSON.

Is it meant to go into `capacitor.config.ts`? 

Does it also need to go into `capacitor.config.json` in correct JSON format?

**Comment by jepiqueau** [comment]
@bluepuma77 it was capacitor.config.json and it is now capacitor.config.ts i will have to update the README. Thznks for pointing it to me

**Comment by jepiqueau** [comment]
@bluepuma77 fix in v5.6.2 Thanks
